README.md

```diff
@@ -47,3 +30,200 @@ terms of [OCA](http://oracle.com/technetwork/goto/oca).
 - [METRO project](https://javaee.github.io/metro)
 - [JSR 222](https://jcp.org/en/jsr/detail?id=222)
 - [Download standalone distribution](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/2.3.0/jaxb-ri-2.3.0.zip)
+
+## Build instructions
+JavaSE 11 is required to build JAXB.
```

Comment:
also say which maven version is required. Note that if the repo resides in a directory with space in path, then the build fails,... so would be good to either document or fix this ;-)

Reply:
Not sure how does spaces in paths relate to jaxb. Are there any issues regarding that?

Reply:
tests in XJC (?) are failing

Reply:
This is fixed in another PR.

Reply:
the fact that the path XJC is trying to resolve and fails contains '%20'?

Reply:
Can you link the issue?

Reply:
there is none yet

Reply:
#1304 not that hard to spot and/or reproduce

Reply:
Lets fix that in another PR. JAXB repository doesn't contain such spaces in directory names and so won't affect the build by default.